runtime/src/lib.rs

```diff
@@ -254,15 +256,14 @@ construct_runtime!(
 		Staking: staking,
 		Democracy: democracy,
 		Grandpa: grandpa::{Module, Call, Storage, Config<T>, Log(), Event<T>},
+		CuratedGrandpa: curated_grandpa::{Module, Call, Config<T>, Storage},
```

Comment:
If we are using `curated_grandpa`, we should remove `grandpa::SyncedAuthorities` from `OnSessionChange`.